src/50_Visual_Effects/Snackbar_%2F_Toast.html

```diff
@@ -0,0 +1,107 @@
+<!---{
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  [Snackbars](https://material.io/guidelines/components/snackbars-toasts.html) provide brief feedback about an operation through a message at the bottom of the screen.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/../visual_effects/snackbar/toast/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!-- We use amp-bind to trigger the Snackbar...  -->
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <!-- ... and amp-animation to show and hide the Snackbar.  -->
+  <script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+<style amp-custom>
+    .trigger {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+    .trigger > * {
+      padding: 0.5rem;
+      margin: 0.5rem;
+    }
+    .snackbar {
+      position: fixed;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      width: 100%;
+      height: 48px;
+      margin: 0 auto;
+      padding: 14px 24px;
+      max-width: 568px;
+      background: #323232;
+      color: white;
+      border-radius: 2px;
+      font-size: 14px;
+      display: flex;
+      align-items: center;
+      justify-content: flex-start;
+      transform: translateY(100%);
+      z-index: 1000;
+    }
+
+  </style>
+</head>
+<body>
+
+  <!-- ## The Animation -->
+  <!--
+    Our snackbar should dissapear after a few seconds. This could be easily accomplished by adding a CSS class using `amp-bind` to show the snackbar and then using CSS animations to hide the snackbar after a few seconds. However, with this approach it's not possible to trigger the snackbar multiple times (the newly added CSS class won't dissappear). 
```

Comment:
would that clear to start with "We want to show a snackbar that disappears after a few seconds" instead of "Our snackbar.."

Reply:
👍